tests/framework/installer/ceph_manifests_v0.8.go

```diff
@@ -504,6 +504,7 @@ kind: StorageClass
 metadata:
    name: ` + storageClassName + `
 provisioner: ceph.rook.io/block
+reclaimPolicy: ` + reclaimPolicy + `
```

Comment:
@travisn then, how do we set the `reclaimPolicy` for the `storageclass`?

Reply:
The policy in this file will only impact the upgrade_test.go, which you're not using. Since you're setting the policy in `ceph_manifests.go`, your tests will see the new policy.